### DISCUSSION TITLE
A feature to disable all images for playing in public?

### DISCUSSION THREAD

**Discussion by LOKSTED** [ppy/osu#16871] [Ideas] [answered]
During play, this can be solved by dimming the background but in the song select screen, there is no way to remove the preview background images. While there aren't any nsfw images, there are uncomfortable amount of underage anime girls that the map makers put in, sometimes doing pretty ඞsy poses.

The solution could be a switch in the settings to disable all preview images in the selection screen

**Comment by integerrr** [comment, marked as answer]
#13333, see peppy's comment
